legacy/src/main/java/org/opensearch/sql/legacy/executor/join/ElasticJoinExecutor.java

```diff
@@ -256,23 +261,44 @@ protected void updateMetaSearchResults(SearchResponse searchResponse) {
     this.metaResults.updateTimeOut(searchResponse.isTimedOut());
   }
 
-  protected SearchResponse scrollOneTimeWithMax(
-      Client client, TableInJoinRequestBuilder tableRequest) {
-    SearchRequestBuilder scrollRequest =
-        tableRequest
-            .getRequestBuilder()
-            .setScroll(new TimeValue(60000))
-            .setSize(MAX_RESULTS_ON_ONE_FETCH);
+  public SearchResponse getResponseWithHits(
```

Comment:
For all executor changes around this, do you know if we have enough IT to cover?

Reply:
We have total 68 IT for join, 5 are ignored. Test covers setting scroll page size to 2 to test scrolling which will replaced by search_after with pit call so should be good but I am not sure how to check if it's enough